Dockerfile

```diff
@@ -54,8 +69,8 @@ RUN poetry install --no-dev --no-interaction --no-ansi --no-root
 
 COPY . /app
 
-EXPOSE 8080
+EXPOSE 80
 
 ENV PYTHONPATH "${PYTHONPATH}:${WORKDIR}/app/api"
 
-CMD ["gunicorn", "main:app", "--workers", "2", "--threads", "1", "--bind", "0.0.0.0:8080", "-k", "uvicorn.workers.UvicornWorker", "--timeout", "250"]
+CMD ["/usr/bin/supervisord", "-c", "/etc/supervisor/supervisord.conf"]
```

Comment:
Does supervisor start the app?

Reply:
ahem, I missed the supervisor.conf file; sorry about that.